### DISCUSSION TITLE
checkpoint for InPaint mode ?

### DISCUSSION THREAD

**Discussion by goubouche** [danilodelucio/NukeDiffusion#3] [General]
Hi ! First, thanks for your work, that works really well ! But is there a reason why checkpoints aren't avalaible when using Inpaint mode ?

**Comment by danilodelucio** [comment]
Hey, thanks a lot for the compliment, I'm glad that it is working and you are enjoying the tool! 😍

I didn't find the right methods to indicate the local checkpoints. I sent this question to the Hugging Face repository but haven't received the right answer yet.

Once I figure out how to use a custom checkpoint I will update the tool.
Thanks!
